### DISCUSSION TITLE
Getting the word count for a markdown field

### DISCUSSION THREAD

**Discussion by stephan-v** [statamic/cms#5694] [Q&A] [answered]
Is there a way to get the current word count for a markdown field or perhaps a bard fieldtype in the editor?

When writing a blog post it would be nice to have this instead of going back and forth between something to paste my content in to check the amount of words.

**Comment by ryanmitchell** [comment]
There is a word_count modifier - did you look into that? https://statamic.dev/modifiers/word_count

**Reply by duncanmcclean** [reply]
I think @stephan-v is talking about having a word count in the CP while editing, rather than on the front-end.

**Reply by ryanmitchell** [reply]
You're right - sorry!

**Comment by jasonvarga** [comment, marked as answer]
Not at the moment, but you can open a feature request. https://github.com/statamic/ideas
